plugins/workspace-plugin/package.json

```diff
@@ -18,6 +18,9 @@
     "@theia/plugin": "next",
     "@theia/plugin-packager": "latest"
   },
+  "extensionDependencies": [
```

Comment:
Sorry, I didn't get why the Workspace Plug-in has to depend on it?
Could you please elaborate a bit?

If it's just for bootstrapping, isn't the `*` activation event enough?

Reply:
It's to make sure that it's loaded as first plug-ins because there is no concept of optional dependency in VScode api